### ISSUE TITLE
Replace bfraser/grafana with voxpupuli/grafana

### ISSUE THREAD

**Issue by dnsmichi** [Icinga/icinga-vagrant#55]
https://github.com/voxpupuli/puppet-grafana - see bfraser/puppet-grafana#116

**Comment by dnsmichi** [comment]
Depends on #56 since the module release 3.0.0 requires Puppet 4.x.

**Comment by dnsmichi** [comment]
Already done.
